dali/operators/reader/coco_reader_op.cc

```diff
@@ -36,9 +36,11 @@ Tensor (``m * category_id``).)code")
   .AddOptionalArg("shuffle_after_epoch",
       R"code(If set to True, the reader shuffles the entire  dataset after each epoch.)code",
       false)
-  .AddArg("file_root",
-      R"code(Path to a directory that contains the data files.)code",
-      DALI_STRING)
+  .AddOptionalArg<string>("file_root",
```

Comment:
you can undo changes in this file, as I will be reworking this reader a little bit soon

Reply:
I would keep it as Michal did the changes. I think you can work on top of @mzient PR.